internal/lib/remove.go

```diff
@@ -9,45 +9,28 @@ import (
 	"github.com/pkg/errors"
 )
 
-// Remove removes a container
-func (c *ContainerServer) Remove(ctx context.Context, container string, force bool) (string, error) {
-	ctr, err := c.LookupContainer(container)
-	if err != nil {
-		return "", err
-	}
+// RemoveAndDeleteContainer performs the steps needed to delete a container's storage and from the runtime,
+// and removes it from the container server
+func (c *ContainerServer) RemoveAndDeleteContainer(ctx context.Context, ctr *oci.Container) error {
 	ctrID := ctr.ID()
 
-	cStatus := ctr.State()
-	switch cStatus.Status {
-	case oci.ContainerStatePaused:
-		return "", errors.Errorf("cannot remove paused container %s", ctrID)
-	case oci.ContainerStateCreated, oci.ContainerStateRunning:
-		if force {
-			if err = c.StopContainer(ctx, ctr, 10); err != nil {
-				return "", errors.Wrapf(err, "unable to stop container %s", ctrID)
-			}
-		} else {
-			return "", errors.Errorf("cannot remove running container %s", ctrID)
-		}
-	}
-
 	if err := c.runtime.DeleteContainer(ctx, ctr); err != nil {
```

Comment:
Are these tests still done elsewhere now?

Reply:
yeah they're done inside of the runtime with  calls to "ShouldBeStopped"